d.configuration.md

```diff
@@ -358,12 +358,18 @@ kubectl get secret mysecret2 -o yaml
 echo YWRtaW4K | base64 -d # on MAC it is -D, which decodes the value and shows 'admin'
 ```
 
-Alternative:
+Alternative using `--jsonpath`:
 
 ```bash
 kubectl get secret mysecret2 -o jsonpath='{.data.username}{"\n"}' | base64 -d  # on MAC it is -D
 ```
 
+Alternative using `--template`:
+
+```bash
+kubectl get secret mysecret2 --template '{{.data.username}}' | base64 -d  # on MAC it is -D
```

Comment:
Is this MacOS?

Reply:
@dgkanatsios 

On Linux. I've added a comment as follows to indicate that you should pass different flag (`base64 -D`).

```
# on MAC it is -D
```

Reply:
Thanks!